telco-hub/configuration/example-overlays-config/acm/options-agentserviceconfig-patch.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# patching mco StorageClass
+
+- op: replace
+  path: /spec/databaseStorage/storageClassName
+  value: "ocs-storagecluster-cephfs"  # filesystem StorageClass
+
+- op: replace
+  path: /spec/filesystemStorage/storageClassName
+  value: "ocs-storagecluster-cephfs"  # filesystem StorageClass
+
+- op: replace
+  path: /spec/imageStorage/storageClassName
+  value: "ocs-storagecluster-cephfs"  # filesystem StorageClass
+
+  # Configure the osImages urls.
+  # When disconnected, the urls should point to a mirrored registry.
+- op: replace
+  path: "/spec/osImages"
+  value:
+    - cpuArchitecture: x86_64
+      openshiftVersion: "4.16"
+      rootFSUrl: https://mirror.example.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.16/latest/rhcos-live-rootfs.x86_64.img
+      url: https://mirror.example.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.16/latest/rhcos-live.x86_64.iso
+      version: 416.94.202411261619-0
+    - cpuArchitecture: "x86_64"
+      openshiftVersion: "4.17"
+      rootFSUrl: https://mirror.example.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.17/latest/rhcos-live-rootfs.x86_64.img
+      url: https://mirror.example.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.17/latest/rhcos-live.x86_64.iso
+      version: "417.94.202409121747-0"
+    - cpuArchitecture: x86_64
+      openshiftVersion: "4.18"
+      rootFSUrl: https://mirror.example.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.18/latest/rhcos-live-rootfs.x86_64.img
+      url: https://mirror.example.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.18/latest/rhcos-live.x86_64.iso
+      version: 418.94.202502100215-0
+
+# when disconnected, the spoke clusters will need to use also a mirrored registry.
+# [ToDo] To test, implement and include and example how to customize this registry
+
+# when connected enviroment (or until we close the previous ToDo) we dont need
+# the spokes to use a mirrored registry. So, AgentServiceConfig dont need to configure the registry.
+# [ToDo] The next remove should be commented by default, because disconnected is the default.
+- op: remove
+  path: /spec/mirrorRegistryRef
```

Comment:
The ToDo is correct. By default this should be included.

Reply:
you are right. Fixed.